openfisca_core/holders.py

```diff
@@ -7,31 +7,29 @@
 
 from . import periods
 from .commons import empty_clone
+from .columns import MONTH, YEAR, PERMANENT
 
 
 class DatedHolder(object):
-    """A view of an holder, for a given period (and possibly a given set of extra parameters).
-    If the variable is not cached, it also contains the value of the variable for the given date."""
+    """The value of a variable for given a period (and possibly a given set of extra parameters).
```

Comment:
You replaced "for a given period" by "for given a period" but I think the original was the right way.